alvr/client_openxr/src/extra_extensions/facial_tracking_htc.rs

```diff
@@ -64,6 +85,8 @@ impl FacialTrackerHTC {
             ))?;
 
             if facial_expressions.is_active.into() {
+                weights.set_len(self.expression_count);
```

Comment:
I'm truly surprised, but apparently this is the right way to do ffi with vectors :+1: